.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,31 @@
-<!-- Describe your pull request here -->
+**Describe your pull request here**
```

Comment:
I don't mind having this reminder visual all the time.
But I'm also okay with the HTML comment, if anyone objects.

Reply:
Maybe at least have it not in bold? IMO, it otherwise stands out too much when the PR submitter leaves it as it is.

Reply:
> stands out too much when the PR submitter leaves it as it is.

Right, with the previous `Replace this sentence` wording it was intentional to make it stand out to remind people to change it (not that it worked for everyone, of course...).
But without the telling them to `Replace`, it makes sense to not have it stand out, because it might not get replaced that often.